share/csm_share/test/unit/dynamic_vector/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ include_directories(.)
 
 # Because these tests use type parameterization, unfortunately we need to
 # preprocess *before* running the pFUnit preprocessor, then again *after*.
-if(${CMAKE_C_COMPILER_ID} STREQUAL GNU)
+if(${CMAKE_C_COMPILER_ID} STREQUAL GNU OR ${CMAKE_C_COMPILER_ID} STREQUAL Clang)
```

Comment:
Macs ship with the Clang compiler. I'm not completely positive that it's right to treat Clang like GNU here, but it seemed to work.